flink-connector-examples/doc/flink-wordcount/README.md

```diff
@@ -109,16 +109,16 @@ Submit `WordCountWriter` job
 - Browser to flink-examples/build/install/pravega-flink-examples/lib and select the uber jar file: pravega-flink-examples-0.2.0-SNAPSHOT-**all**.jar
 - Click **Upload** button
 - Click the checkbox next to the uploaded jar file.
-- In the **Entry Class** field, enter: ```io.pravega.examples.flink.wordcount.WordCountWriter```
-- In the **Program Arguments** field, enter: ```--host localhost --port 9999 --controller tcp://localhost:9090```
```

Comment:
Why the second `-` before `host` has been deleted?

Reply:
Sorry. Often my virtual box loses focus. Removing it was inadvertent. Now fixed

Reply:
I changed the image to reflect the package change. Must have lost the change somewhere in the windows to ubuntu exchange. Updated the image and found a couple of more lines with the old package name on it. Also noticed the intellij image had the old package name. My intellij interface for  run configurations looks slightly different from yours, but I'll change the image out with the snapshot from my intellij if you like.